src/core/annotation.js

```diff
@@ -332,6 +332,16 @@ class Annotation {
       this._streams.push(this.appearance);
     }
 
+    const kids = dict.getRaw("Kids");
+    const kidsId = [];
```

Comment:
Can we please initialize this to `null` instead here, and add `if (!kidsId) { kidsId = []; }` before pushing below, to avoid having to include an empty Array in the data for *every* single Annotation?